Simplenote/Classes/UIColor+Studio.swift

```diff
@@ -0,0 +1,91 @@
+import Foundation
+
+
+// MARK: -
+//
+extension UIColor {
+
+    /// Initializes a new UIColor instance with a given ColorStudio value
+    ///
+    convenience init(studioColor: ColorStudio) {
+        self.init(hexString: studioColor.rawValue)
+    }
+
+    /// Initializes a new UIColor instance with a given ColorStudio Dark / Light set.
+    /// Note: in `iOS <13` this method will always return a UIColor matching the `Current` Interface mode
+    ///
+    convenience init(lightColor: ColorStudio, darkColor: ColorStudio) {
+        guard #available(iOS 13.0, *) else {
+            let targetColor = SPUserInterface.isDark ? lightColor : darkColor
```

Comment:
Is this correct? You return the **light color** if `SPUserInterface.isDark`

Reply:
Mother of god **THANK YOU**. It didn't "break" before ... because in this PR there are no colors with Light / Dark!!!

Thank you!!!